<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminView</class>
 <widget class="QWidget" name="AdminView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>600</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_restSelect">
    <widget class="QWidget" name="widget_restListAvailable" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>201</width>
       <height>481</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_confirmRestChanges">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>520</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm changes</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_restListDeleted" native="true">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>201</width>
       <height>481</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_restListAvailable">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Available Restaurants</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_restListDeleted">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Hidden Restaurants</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>540</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Drag restaurants here to hide</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_addFromFile">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add restaurants from file...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>540</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Drag restaurants here to show</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_editMenu">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit menu items...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_menuSelect">
    <widget class="QPushButton" name="pushButton_selectRest">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Go back...</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_price">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="prefix">
      <string>$</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_menuAvailable" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>201</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="widget_menuDeleted" native="true">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>201</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_menuDeleted">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Hidden Menu Items</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_menuAvailable">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Available Menu Items</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
